### PR TITLE
Issue 1683 - Add Zondax libraries

### PR DESCRIPTION
**Pull request by ElPaisano**
Addresses https://github.com/filecoin-project/filecoin-docs/issues/1683

I used the info in https://github.com/filecoin-project/testnet-hyperspace#resources -> Filecoin Solidity libs. Based off that, looks like there's two Zondax libraries, Filecoin Solidity and FEVM Precompiles. Not sure if I misunderstood that

I should note that I am keeping descriptions intentionally short because it looks like both of these libraries are very much under development, so I'm just trying to give the reader the high level and then send them to the maintainers docs. Not sure if that fits the bill here, but that would be my recommendation 

*Blocker* 
The info on FEVM Precompiles is a bit hard to fill out based on https://github.com/Zondax/fevm-solidity-precompiles/tree/main/docs/fevm-solidity-precompiles... I'm really not clear what the status of this project is, as there README isn't even complete and there `docs` folder is hard to parse...

### CODE REVIEW

**Comment by ElPaisano** [comment]
Hey @johnnymatthews can you take a look at my comments in the description and lmk what you think?

File: content/en/developers/infrastructure/libraries/index.md [hunk 1]
```diff
@@ -15,3 +15,19 @@ aliases:
 ---
 
 {{< beta-warning >}}
+
+## Zondax 
+
+Zondax provides the [Filecoin Solidity npm package](#filecoin-solidity) and the [FEVM Precompiles library](#fevm-precompiles).
+
```
Comment (johnnymatthews) [inline content/en/developers/infrastructure/libraries/index.md:22]: I'd remove this specific callout to Zondax, and add a simple sentence to the two sections below just saying _"Hey this is made by Zondax, here's a link to their website"_ -- or something to that effect.

File: content/en/developers/infrastructure/libraries/index.md [hunk 2]
```diff
@@ -15,3 +15,19 @@ aliases:
...
+### Filecoin Solidity
+
+{{< alert >}}
+The Filecoin Solidity project is [currently in beta](https://docs.zondax.ch/fevm/filecoin-solidity#disclaimer-%EF%B8%8F%EF%B8%8F).
+{{< /alert >}}
+
+The _Filecoin Solidity_ npm package provides a set of Solidity libraries that allows Solidity smart contracts to seamlessly call Filecoin built-in actors methods, and perform cross-platform calls to the real Filecoin built-in actors. Additionally, a set of mock libraries that respond to specific scenarios is available. The scenarios are based on the received parameters instead of real calls. For further information, including information on how to use the package, see the [official documentation](https://docs.zondax.ch/fevm/filecoin-solidity/).
```
Comment (johnnymatthews) [inline content/en/developers/infrastructure/libraries/index.md:29]:
```suggestion
_Filecoin.sol_ provides a set of libraries that allows Solidity smart contracts to seamlessly call built-in actors methods. Additionally, a set of mock libraries that respond to specific scenarios is available. The scenarios are based on the received parameters instead of real calls. For further information, including information on how to use the package, see the [official documentation](https://docs.zondax.ch/fevm/filecoin-solidity/).
```

**Review by johnnymatthews — CHANGES_REQUESTED** [review]

File: content/en/developers/infrastructure/libraries/index.md [hunk 3]
```diff
@@ -15,3 +15,19 @@ aliases:
 ---
 
 {{< beta-warning >}}
+
+## Zondax 
+
+Zondax provides the [Filecoin Solidity npm package](#filecoin-solidity) and the [FEVM Precompiles library](#fevm-precompiles).
+
+## Filecoin.sol
```
Comment (longfeiWan9) [inline content/en/developers/infrastructure/libraries/index.md:23]:
```suggestion
## Filecoin.solidity
```

File: content/en/developers/infrastructure/libraries/index.md [hunk 4]
```diff
@@ -15,3 +15,19 @@ aliases:
...
+
+{{< alert >}}
+The Filecoin Solidity project is [currently in beta](https://docs.zondax.ch/fevm/filecoin-solidity#disclaimer-%EF%B8%8F%EF%B8%8F).
+{{< /alert >}}
+
+_Filecoin.sol_ provides a set of libraries that allows Solidity smart contracts to seamlessly call built-in actors methods. Additionally, a set of mock libraries that respond to specific scenarios is available. The scenarios are based on the received parameters instead of real calls. For further information, including information on how to use the package, see the [official documentation](https://docs.zondax.ch/fevm/filecoin-solidity/).
```
Comment (longfeiWan9) [inline content/en/developers/infrastructure/libraries/index.md:29]: 1. Let's be clear that filecoin.solidity allows calling exported built-in actor methods, not all of them.  Some community devs complained to me that filecoin.solidity only provides a few built-in actors calls which is a misunderstanding. 
2. mock version of the libraries is not recommended to use anymore. It was only for supporting FVM hack in wallaby since there was no real deal data. 
3. we should also provide link to their [GitHub](https://github.com/Zondax/filecoin-solidity).

**Comment by longfeiWan9** [comment]
I think there are two more tools worth mentioning here.

- [Filecoin Signing tools](https://github.com/Zondax/filecoin-signing-tools)
- [Filecoin Address from Glif](https://github.com/glifio/modules/tree/primary/packages/filecoin-address), I noticed this has been used a lot by devs.

File: content/en/developers/infrastructure/libraries/index.md (hunk 4 above)
Comment (ElPaisano) [reply content/en/developers/infrastructure/libraries/index.md]: @longfeiWan9 

1. Got it, added copy for this
1. Got it, removed mention of mock version
1. Got it, added link

File: content/en/developers/infrastructure/libraries/index.md [hunk 5]
```diff
@@ -15,3 +15,19 @@ aliases:
...
+
+#### FEVM Precompiles
```
[the remark this answers is not in the document]
Comment (ElPaisano) [reply content/en/developers/infrastructure/libraries/index.md]: @longfeiWan9 went ahead and removed it for now. We can add back into this page if it is confirmed whether the tool is being developed or not

**Comment by ElPaisano** [comment]
> I think there are two more tools worth mentioning here.
> 
> * [Filecoin Signing tools](https://github.com/Zondax/filecoin-signing-tools)
> * [Filecoin Address from Glif](https://github.com/glifio/modules/tree/primary/packages/filecoin-address), I noticed this has been used a lot by devs.

@longfeiWan9 got it, added in blurbs about these tools

**Comment by ElPaisano** [comment]
Also, the https://docs.zondax.ch/docs/filecoin/filecoin-signing-tools/ official doc link is 404ing, anyone know if I should look elsewhere or is it just a one time thing

File: content/en/developers/infrastructure/libraries/index.md (hunk 5 above)
[the remark this answers is not in the document]
Comment (longfeiWan9) [reply content/en/developers/infrastructure/libraries/index.md]: I also confirmed with the Zondax team. This FEVM precompiles was a proof of concept thing which will not be work on anymore.

**Comment by longfeiWan9** [comment]
The new docs site is https://docs.zondax.ch/filecoin-signing-tools/

**Review by johnnymatthews — APPROVED** [review]
Looks good from my end.
